docs/examples/clojure-example.md

```diff
@@ -0,0 +1,10 @@
+<link rel="stylesheet" type="text/css" href="https://storage.googleapis.com/app.klipse.tech/css/codemirror.css" />
+
+# sample Clojure code
```

Comment:
@lambda-knight 
Sorry for the late reply.
Can you share the cli command you run in order to generate the html file?

I tried to use `marked` but it generates only the body of the html

```bash
$ marked docs/examples/clojure-example.md
<link rel="stylesheet" type="text/css" href="https://storage.googleapis.com/app.klipse.tech/css/codemirror.css" />

<h1 id="sample-clojure-code">sample Clojure code</h1>
<pre><code class="language-clj">(map inc &#39;(1 2 3 4 5 6 7 8 9 10))</code></pre>
<script type="text/javascript">window.klipse_settings = {selector: '.language-clj'};</script>
<script src="https://storage.googleapis.com/app.klipse.tech/plugin/js/klipse_plugin.js"></script>
```

Reply:
@lambda-knight @MageMasher  Can you share the cli command you run in order to generate the html file?